seq_tools/validation/c609_fastq_sanity.py

```diff
@@ -147,9 +145,9 @@ def fastq_test_regex(fastq,path):
             return False,"Line #%s within FASTQ file %s not following FASTQ format, missing '+' at start of the line" % (line_tracker+1,fastq)
         elif line_count%4==3 \
             and \
-            re.findall(r'[^'+''.join([chr(phred+64) for phred in range(0,41)])+']',line) \
+            re.findall('%r'%'[^'+r''.join([chr(phred+64) for phred in range(0,41)])+']',line) \
             and \
-            re.findall(r'[^'+''.join([chr(phred+33) for phred in range(0,42)])+']',line):
+            re.findall('%r'%'[^'+r''.join([chr(phred+33) for phred in range(0,42)])+']',line) :
```

Comment:
This returns `'[^'!"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJ]` of the pattern, is it correct?

Reply:
This PR is mainly for c685. Do you mind to move fixes to c609 into separate PR?

Reply:
https://github.com/icgc-argo/seq-tools/pull/99#discussion_r997424515
Should be:
`'[^!"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJ]'` (missing backslash).
The change was b/c the formatting would result in the `\` being treated as an escape character and not looking for `\`. I tried doing `\\` but that didn't work either. Testable via:
```
>>>line='AGGH\!'
>>> re.findall(r'[^'+''.join([chr(phred+33) for phred in range(0,42)])+']',line)
['\\']
>>> re.findall('%r'%'[^'+r''.join([chr(phred+33) for phred in range(0,42)])+']',line)
[]
```
https://github.com/icgc-argo/seq-tools/pull/99#discussion_r997426308
Sure. Sorry was lazy